scrapinghub/spidermon
# Add SNS Action for Spidermon Notifications

### Summary:
This pull request introduces a new action that allows Spidermon to send notifications via AWS Simple Notification Service (SNS). This provides users with an alternative way to receive notifications and integrate Spidermon with other AWS services or third-party applications through SNS.

### Changes:
- Added `SendSNSNotification` class to handle SNS notifications.
- Provided documentation on how to set up and use the SNS action.

### Benefits:
- Offers more flexibility for users who are heavily invested in the AWS ecosystem.
- Allows for easy integration with other AWS services like Lambda, SQS, etc., by leveraging SNS topics.

### How to Test:
1. Set up AWS credentials and an SNS topic as described in the documentation.
2. Configure Spidermon to use the new SNS action.
3. Trigger a Spidermon notification to see it being sent to the specified SNS topic.

Looking forward to feedback and any additional changes required.
